draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1354,26 +1340,24 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
-
-This example illustrates how to use the artifacts defined in this specification to secure a payload
-that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
-and does not use JSON-LD to provide semantic definitions for claims. The example uses a content type `credential-claims-set+json` defined in [@VC_JWT], Section 3 in a `cty` JOSE Header value.
+## Example 4a - Verifiable Credentials with JSON payload based on the SD-JWT format
 
-SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a W3C VC.
+This example illustrates how to use the artifacts defined in this specification to secure
+Verifiable Credentials with JSON payload based on the SD-JWT format as defined in [@I-D.terbu-sd-jwt-vc].
 
-A Holder-signed Verifiable Presentation as defined in [@VC_DATA_v2.0] is equivalent to
-an SD-JWT Presentation with a Key Binding JWT.
+The example uses a media type `vc+sd-jwt`.
```

Comment:
```suggestion
The example uses a media type `vc+sd-jwt` as the `typ` JOSE header parameter value.
```
per torsten's review

Reply:
The problem with saying that is that the example doesn't actually have a `typ` JOSE header parameter at all. Saying just, "The example uses a media type `vc+sd-jwt`" felt less outright wrong. But it's not really correct either. I think these examples might be better off just not saying anything about `typ` or `cty` headers or media types. 

```suggestion
```

Reply:
i would prefer to keep this because sd-jwt vc mandates using this typ.
I can add a header manually just like for KB JWT - torsten asked about the header too.